stream-compositions/services/legal-entity-composition-service/src/main/java/com/backbase/stream/compositions/legalentity/Application.java

```diff
@@ -0,0 +1,16 @@
+package com.backbase.stream.compositions.legalentity;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.web.servlet.support.SpringBootServletInitializer;
+import org.springframework.cloud.client.discovery.EnableDiscoveryClient;
+import org.springframework.context.annotation.ComponentScan;
+
+@EnableDiscoveryClient
+@SpringBootApplication
+@ComponentScan("com.backbase.stream")
+public class Application extends SpringBootServletInitializer {
```

Comment:
let's try to use unique names for all applications: e.g. `LegalentityCompositionApplication`